src/components/indicator_table_body.tsx

```diff
@@ -50,7 +50,7 @@ function TABLE_BODY(props: RegisterProps) {
       return null;
     });
     return register_block;
-  });
+  }, done);
```

Comment:
The second argument to map is the value to use as `this` inside the callback. Not needed here.

```suggestion
  });
```